src/services/ban_service.py

```diff
@@ -3,66 +3,62 @@
 from typing import Optional, Tuple
 from util import log_info_and_embed, send_dm
 from settings import get_settings
+from datetime import datetime, timedelta, timezone
 
 settings = get_settings()
 logger = logging.getLogger(__name__)
 
 
 async def ban_user(
-    user: discord.Member, reason: str
-) -> Optional[Tuple[bool, bool, str]]:
+    user: discord.Member, reason: str, delete_messages: bool
+) -> Optional[Tuple[bool, str]]:
     """Executes ban of user.
 
-    Args:
-        user (discord.Member): mention or "@"/user ID of the user being banned.
-        reason (str): description of ban reason. Length of reason must be less than 512 characters
+     Args:
+         invoking_member (discord.Member): The moderator initiating the ban.
```

Comment:
nit: we should get rid of this description if we're not passing this in anymore

Reply:
done